upd_script/antenna_wifi_override.conf

```diff
@@ -0,0 +1,9 @@
+[Unit]
+[Unit]
+Description=R4R Antenna Wifi Indicator 
+Requires=novnc.service
+After=novnc.service
+
+[Service]
+ExecStart=
```

Comment:
Likewise for `ExecStart`. Only one is required.

Reply:
Not based on my testing and what I read. If you just have one, then the ExecStart is additive and then it fails. 
The first one is meant to cancel the one in the other service.

Reply:
See https://askubuntu.com/questions/659267/how-do-i-override-or-configure-systemd-services
_I had to explicitly clear ExecStart before setting it again, as it is an additive setting, similar to After, Environment (as a whole, not per-variable) and EnvironmentFile, and opposed to overriding settings like RestartSec or Type. ExecStart can have multiple entries only for Type=oneshot services._

My testing was giving me exactly that behaviour